<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SimpleChatTab</class>
 <widget class="QWidget" name="SimpleChatTab">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="lblStatus">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Status:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="lblDescription">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Description:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="ChatWidget" name="chatWidget"/>
   </item>
   <item>
    <widget class="BetterPlainTextEdit" name="edtInput">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>60</height>
      </size>
     </property>
     <property name="tabChangesFocus">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="btnInputSend">
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>23</height>
      </size>
     </property>
     <property name="text">
      <string>Send</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="btnSendImage">
     <property name="text">
      <string>Send Image...</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ChatWidget</class>
   <extends>QScrollArea</extends>
   <header>widgets/ChatWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>BetterPlainTextEdit</class>
   <extends>QPlainTextEdit</extends>
   <header>widgets/BetterPlainTextEdit.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
